Add Firefox

trockenasche opened the pull request:
Firefox account for Firefox Sync and/or addons.mozilla.org

trockenasche committed:
Add Pushover

Pushover is Push Notifications for Android, iOS, and Desktop with API support

trockenasche committed:
Merge pull request #1 from 2factorauth/master

make even

trockenasche committed:
Merge pull request #2 from 2factorauth/master

Merg origin

trockenasche committed:
add Firefox

Firefox account for Firefox Sync and/or addons.mozilla.org

mxxcon commented on _data/other.yml:
```diff
@@ -222,6 +222,13 @@ websites:
       doc: https://www.findmyshift.com/help/what-is-two-factor-authentication
 
+    - name: Firefox
+      url: https://accounts.firefox.com/settings/two_step_authentication
```
This url should point either to the main firefox.com or at least accounts.firefox.com, but not settings page.

trockenasche replied:
Okay, thanks, I will do so.

trockenasche committed:
update Firefox URL

as desired from https://github.com/2factorauth/twofactorauth/pull/3245#pullrequestreview-126492438
